src/cmds/list:imported.ts

```diff
@@ -35,6 +35,7 @@ const entityName: {
   github: 'repo',
   'github-enterprise': 'repo',
   'bitbucket-cloud': 'repo',
+  'gcr': 'images'
```

Comment:
could you update the docs in readme to list this as supported too: https://github.com/snyk-tech-services/snyk-api-import/blob/master/docs/import.md#to-skip-all-previously-imported-targets